src/main/resources/static/api.yml

```diff
@@ -399,6 +406,124 @@ paths:
         404:
           description: No offering has the supplied id (Not Found)
 
+  /course-participation-history:
+    post:
+      tags:
+        - Course participation history
+      summary: Record information about a person's prior participation in a course that was not managed by this service
+      requestBody:
+        required: true
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/CreateCourseParticipation'
+      responses:
+        200:
```

Comment:
Sorry, just spotted this - should this be a 201 if we're creating an entity here?

Reply:
Yes. You're right.